### DISCUSSION TITLE
Replace "-" by value

### DISCUSSION THREAD

**Discussion by Samohtyve** [5axes/ProfilAnalyser#8] [Ideas]
Hi, 
First of all, thank you for this great work allowing us to better work on our profiles and to improve them.

We can see that when the value is the original one, there is this line ("-"). Would it be possible to replace the "-" with the values in order to better measure the difference?
Best regards,

**Comment by 5axes** [comment]
Hello,
Thanks for your question. It's not really possible, because if you have a "No value" parameter it's because this parameter is not set in the profile. So it's also an information by itself. No value means this value will be not modified by the profile and in the analyse we need to know it.   We could have a solution of color or anything with the global value but it could be very confusing for the user so for the moment it's not a solution desirable in the plugin. But if you have any solution to keep this "none information" easily understandable or visible, I could reconsidered my point of view.
